### DISCUSSION TITLE
Bandwidth Question (IPSEC tunnel)

### DISCUSSION THREAD

**Discussion by dlee35** [Security-Onion-Solutions/securityonion#8103] [Unsupported Versions]
Hello Onionistas!

I have a quick question for the community at large. As pedantic as this may sound, a colleague and I have recently gotten into a debate about potential deployment scenarios regarding which of those use more bandwidth versus the feasibility/usability of either example. Security Onion's documentation states that the use of a Heavy Node is not recommended in most situations, and I am also aware that using a single Heavy Node downstream of the Manager is not something typically condoned as it's more beneficial to scale these resources in support of a grid deployment. However, this question is specifically directed at the following deployment scenarios below.

Which of these *two* strategies do you believe would **use more bandwidth** across the IPSEC tunnel depicted in the diagram? _Why_?   
- A single, standalone Security Onion server deployed in the remote network with connectivity to Hunt, Kibana, etc.
- A Manager node is available locally with a single Heavy Node deployed to the remote site.

**Facts**:
- IPSEC tunnels in both scenarios are of the exact same configuration, bandwidth, etc.
- Network routes and hops are equal in comparison
- Security Onion versions are exactly the same
- Monitored networks will be exactly the same
- The number of analysts querying is equal
- Analysts will use the same browsers
- The queries levied are the exact same
- The count of records returned is... you guessed it, the same

**Assumptions**:
- Resources will be split based upon manager/heavy node allocation vs standalone (ex: 2x 8-CPU 16GB vs 1x 16-CPU 32GB)
- IPSEC tunnel will remain reasonably available throughout testing
- Salt traffic will not be a consideration (even though it will utilize bandwidth)

**Diagram**:

![image](https://user-images.githubusercontent.com/7849311/172896002-b399dd3b-d295-4eba-bb67-42115279543d.png)

Thanks in advance for reading, entertaining, and (_potentially_) answering!

**Comment by TOoSmOotH** [comment]
@dlee35 Option 1 would use more bandwidth in theory since the heavy node will check into the manager every 15 minutes. Most of it is cached so it wouldn't be a huge difference. Where the real difference in usage would come would be during a soup when the HN downloads its containers from the manager. You also have telegraf data being pushed to influxdb from the HN to the manager. Not a ton of data but if we are splitting hairs then technically it would generate more bandwidth.
